### ISSUE TITLE
NVDA Reporting  Contents of Combo Boxes with High Verbosity

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2685]
**Reported by ateu on 2012-09-21 15:48**
Steps to reproduce:
1. Open Google Chrome Canary or a newer version of Chrome;
2. Go to the NVDA ticket editor;
3. Access any combo box and use the arrow keys to hear the items.

Expected result: When arrowing through the items, NVDA should announce just the current item.
Actual result: NVDA announces the previous item, list item editable and the current item. (I.g. in the combo box where we choose the type, when arrowing from the first to the second item, NVDA says "defect list item editable enhancement" instead of just enhancement.

**Comment by ehollig** [comment]
This appears to be fixed in May 2015 from Google Issue [Chromium:323462](https://bugs.chromium.org/p/chromium/issues/detail?id=323462)

**Comment by ehollig** [comment]
Closing, as the Google Issue [Chromium:323462](https://bugs.chromium.org/p/chromium/issues/detail?id=323462) has been fixed.
